### DISCUSSION TITLE
Space character prevents Noop Static Components from resolving assets

### DISCUSSION THREAD

**Discussion by 1800joe** [noop-inc/documentation#64] [Bugs] [answered]
```{"code":"StaticAssetNotFound","message":"Asset /assets/portrait/Foo%20Bar.png not found"}```

The server-timing header suggests that Noop is generating this response:

```
_n11;desc="NoopRouter us-west-2";, _n14;desc="NoopRouterProcessing";dur=0.844, _n15;desc="NoopRouterWaiting";dur=45.842, _n23;desc="NoopStaticProcessing";dur=43.179, _n12;desc="NoopCluster us-west-2";

```

...which makes sense since this is deployed as a static web application.

Overall it reads like the system is trying to find a file with the literal urlencoded %20 in it, instead of translating it to a space.

**Comment by arbitrarytech** [comment, marked as answer]
Resolved and released in Foundation
- https://github.com/noop-inc/foundation/pull/688
